src/transformers/models/speech_to_text/feature_extraction_speech_to_text.py

```diff
@@ -107,15 +110,21 @@ def utterance_cmvn(
             std = np.sqrt(np.maximum(var, 1e-10))
             x = np.divide(x, std)
 
+        if x.shape[0] > input_length:
+            x[input_length:] = padding_value
```

Comment:
@patil-suraj - In the beginning conv layers are used, so IMO it's important to set incorrectly normalized values accordingly

Reply:
Good point, yeah! Just checked in fairseq, padding happens after the processing. So this is important here.